src/Microsoft.TestPlatform.ObjectModel/TestServiceLocator.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+// We don't want this in our shipped code. Build only for debug until I am able to remove it.
+#if DEBUG
+
+#if !NETSTANDARD1_0
+using System;
+#endif
+using System.Collections.Generic;
+
+#pragma warning disable RS0016 // Add public types and members to the declared API
+#pragma warning disable RS0037 // Enable tracking of nullability of reference types in the declared API
+
+public static class TestServiceLocator
```

Comment:
It could, but I would have to change internals visible to, and this won't be built for our release dll, so I will keep it as is, my goal is to be able to get rid of this anyway, because it is one of the things that prevent me from isolating my tests.